source/styleguide/styleguide.11tydata.yml

```diff
@@ -0,0 +1,46 @@
+layout: has_navigation
+navigation: |
+  <nav class="sl-c-list-navigation-wrapper">
+
+  ### [Style Guide](/styleguide)
+
+  - [Brand](/styleguide/brand)
+  - [Color](/styleguide/color)
+  - [Typography](/styleguide/typography)
+  - [Layout](/styleguide/layout)
+  - [Components](/styleguide/components)
+  - [Code](/styleguide/code)
+
+  </nav>
+colors:
+  - name: hopbush
+    hex: 'c69'
+  - name: bouquet
+    hex: 'b37399'
+  - name: venus
+    hex: '998099'
+  - name: patina
+    hex: '699'
+  - name: nebula
+    hex: 'd2e1dd'
+  - name: white
+    hex: 'fff'
+  - name: dawn-pink
+    hex: 'f2ece4'
+  - name: wafer
+    hex: 'e1d7d2'
+  - name: iron
+    hex: 'dadbdf'
+  - name: regent-grey
+    hex: '808c99'
+  - name: pale-sky
+    hex: '6b717f'
+  - name: midnight-blue
+    hex: '036'
+font_families:
+  - style: text
+    value: sans-serif
+  - style: display
+    value: serif
+  - style: code
+    value: monospace
```

Comment:
@stacyk @mirisuzanne Do we want to come up with a different solution for these, so it's not duplicated? It looks like the rest of the items in the old `data/color.yml` and `data/typography.yml` aren't being used, so I only copied over these.

Reply:
@jgerigmeyer I removed the `hex: 'colorvalue'` for each color since this uses the utility class already created/defined in the existing CSS... It helps a bit with the whole duplicate values issue. It is a manual update if they want to change, add or delete a color still, but that is certainly an enhancement we can add if time remains?

Reply:
Added a checklist item to https://trello.com/c/ytD32PgU